src/i18n/fi.json

```diff
@@ -129,6 +130,8 @@
     "validation-stringLengthCounter": " merkkiä jäljellä",
     "validation-requiredInContentLanguages": "Tämä kenttä vaaditaan kaikilla valituilla kielillä",
     "validation-offerIsFreeOrHasPrice": "Maksullinen tapahtuma vaatii hinnan kaikilla valituilla kielillä",
+    "validation-atLeastOneIsTrue": "Valitse vähintään yksi toistopäivä",
+    "validation-isMoreThanOne": "Toistovälin tulee olla vähintään 1 viikko",
```

Comment:
Interesting, that one is not visible on my form. The Lisää kerrat button just keeps the dialog open when clicked. I'm running Chrome 54.0.2840.98. I'll check if this is something like a lurking outdated buggy version of an npm dependency or similar.